index.css

```diff
@@ -302,3 +305,79 @@
   align-items: left;
   font-weight: 700;
 }
+
+.contact_section {
+  padding: 150px 0 20px 0;
+  background-color: #6070ff;
+  border-top-left-radius: 150px;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I'm sorry that this wasn't pointed out by a previous reviewer but I would suggest that you use flexbox to layout your elements. 🎨 This will give you better responsiveness to your page! 🤓 🛠️